yarn.lock

```diff
@@ -220,9 +220,9 @@ acorn@^7.4.0:
   integrity sha512-nQyp0o1/mNdbTO1PO6kHkwSrmgZ0MT/jCCpNiwbUjGoRN4dlBhqJtoQuCnEOKzgTVwg0ZWiCoQy6SxMebQVh8A==
 
 acorn@^8.7.0:
-  version "8.7.0"
```

Comment:
@thyarles 
If we change yarn.lock then there would be change in package.json too
Can you please check